File: GEOMETRIA/1292/main.c
#ifndef M_PI
#define M_PI 3.14159265358979323846
#endif
#include <math.h>
#include <stdio.h>

double largestSide(double f) {
    return f * sin(108 * M_PI / 180) / sin(63 * M_PI / 180);
}

int main() {
    double f;
    while (scanf("%lf", &f) != EOF) {
        printf("%.10lf\n", largestSide(f));
    }
    return 0;
}

File: GEOMETRIA/1292/main.rb
# frozen_string_literal:true

def largest_side(f)
  f * Math.sin(108 * Math::PI / 180) / Math.sin(63 * Math::PI / 180)
end

while f = gets&.to_f
  puts format('%.10f', largest_side(f))
end
